src/rotest/api/resource_control/cleanup_user.py

```diff
@@ -0,0 +1,46 @@
+# pylint: disable=unused-argument, no-self-use
+import httplib
+
+from django.db import transaction
+from swaggapi.api.builder.server.response import Response
+from swaggapi.api.builder.server.request import DjangoRequestView
+
+from rotest.management import ResourceData
+from rotest.management.common.utils import get_username
+from rotest.api.common.responses import CleanupUserResponseModel
+
+
+class CleanupUser(DjangoRequestView):
+    """Cleaning up user's requests and locked resources."""
+    URI = "resources/cleanup_user"
+    DEFAULT_RESPONSES = {
+        httplib.OK: CleanupUserResponseModel,
+    }
+    TAGS = {
+        "post": ["Resources"]
+    }
+
+    def post(self, request, *args, **kwargs):
+        """Cleaning up user's requests and locked resources.
+
+        Args:
```

Comment:
I don't see this argument in the signature

Reply:
Because  it is a new  concept - the models, I wanted to describe what the function  model should actually receive.
We need to  decide what  is the convention  for that